extension.js

```diff
@@ -86,7 +86,8 @@ let excludedApplications = new Array(
 );
 
 const key_bindings = {
-    'show-toggle-tiling': function() { toggleTiling(); }
+    'show-toggle-tiling': function() { toggleTiling(); },
+    'show-toggle-tiling-alt': function() { toggleTiling(); }
```

Comment:
This is the spooky line that somehow is not working on my machine.

Reply:
That works fine on my PC- both <Super>+KP_Enter and <Super>+Return
On my laptop, I do not have keypad, and <Super>+Return works fine (through RDP).
Not sure why you have problem on your PC

Reply:
What do you think if I will change autotile keybinding to Ctrl-[1..9,0]?